pkg/security/mcs/label.go

```diff
@@ -0,0 +1,178 @@
+package mcs
+
+import (
+	"bytes"
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+const maxCategories = 1024
+
+type Labels struct {
```

Comment:
In upstream it is referred to as level in a SecurityContext.  My lexicon understanding is level is a label as is user/type/role.